internal/lvgen/generate.go

```diff
@@ -159,13 +159,45 @@ type Case struct {
 
 // Proc holds information about a libvirt procedure the parser has found.
 type Proc struct {
-	Num        int64  // The libvirt procedure number.
-	Name       string // The name of the go func.
-	LVName     string // The name of the libvirt proc this wraps.
-	Args       []Decl // The contents of the args struct for this procedure.
-	Ret        []Decl // The contents of the ret struct for this procedure.
-	ArgsStruct string // The name of the args struct for this procedure.
-	RetStruct  string // The name of the ret struct for this procedure.
+	Num            int64  // The libvirt procedure number.
+	Name           string // The name of the go func.
+	LVName         string // The name of the libvirt proc this wraps.
+	Args           []Decl // The contents of the args struct for this procedure.
+	Ret            []Decl // The contents of the ret struct for this procedure.
+	ArgsStruct     string // The name of the args struct for this procedure.
+	RetStruct      string // The name of the ret struct for this procedure.
+	ReadStreamIdx  int    // The index of read stream in function argument list
+	WriteStreamIdx int    // The index of read stream in function argument list
+}
+
+// ProcMetaGenerate is a type for flags from @generate annotation
+type ProcMetaGenerate int
+
+// @generate annotation can be none, client, server or both
```

Comment:
This comment doesn't describe what ProcMetaGenerate* do or indicate.

Reply:
These annotations (`@generate` and `@priority`) are not used in our code, only in libvirt, really. I guess I can copy description from `remote_protocol.x` here.